code/__DEFINES/sight.dm

```diff
@@ -1,17 +1,17 @@
 
 #define SEE_INVISIBLE_MINIMUM 5
 

```

Comment:
define renamed because it has nothing to do with observer. comment to explain what it does.